token-lending/js/src/util/layout.ts

```diff
@@ -79,7 +79,7 @@ export const bigInt =
 
         bigIntLayout.decode = (buffer: Buffer, offset: number) => {
             const src = decode(buffer, offset);
-            return toBigIntLE(src);
+            return toBigIntLE(src as Buffer);
```

Comment:
Typescript was giving me an error when I tried running the cli otherwise. It might have to do with the `commonjs` setting though - I'm by no means a typescript expert. Any advice?